apps/frontend/src/libs/hooks/use-popover-position/use-popover-position.hook.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "~/libs/hooks/hooks.js";
+
+type Properties = {
+	left?: number;
+	top?: number;
+};
+
+const usePopoverPosition = <T extends HTMLElement>(
+	reference: React.RefObject<T>,
+	isActive: boolean,
+	usePositioning: boolean = false,
```

Comment:
do we need this flag at all? can't we use the same logic for all popups?
```suggestion
```

Reply:
we use absolute/relative positioning in projects module. If we don't use it, we are allowed to use this hook, but the full width of the project won't be clickable